e2e/configs.ts

```diff
@@ -56,7 +49,7 @@ export const APP_ROUTES = {
 // Sidebar labels
 export const SIDEBAR_LABELS = {
   PERSONAL_FILES: 'Personal Files',
-  FILE_LIBRARIES: 'File Libraries',
```

Comment:
this should probably remain unchanged. This refers to the sidenav link, which is called File Libraries, the parent of My Libraries and Favorite Libraries